docs/user-guide/node-selection/index.md

```diff
@@ -198,3 +198,215 @@ must be satisfied for the pod to schedule onto a node.
 
 For more information on inter-pod affinity/anti-affinity, see the design doc
 [here](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/podaffinity.md).
+
+## Taints and tolerations (beta feature)
+
+### Specifying taints and tolerations
+
+Node affinity, described earlier, allows a pod to "be attracted to" a set of nodes
```

Comment:
Rather than trying to explain Taints and Tolerations as the opposite of Affinity (which assumes that people have read/want to read the Affinity documentation), I suggest starting like this:

"Taints and tolerations are features that work together to ensure that pods are not scheduled onto inappropriate nodes. *Taints* are applied to nodes, and are a way of marking that a node should _not accept_ pods that have a given toleration. *Tolerations* are applied to pods, and specify that those pods should avoid nodes with a given taint.

"